packages/ui/src/ui/containers/ACL/ApproversFilters/ApproversFilters.tsx

```diff
@@ -13,14 +14,23 @@ export default function ApproversFilters() {
     const subjectFilter = useSelector(getApproversSubjectFilter);
 
     return (
-        <Filter
-            placeholder="Filter by subject"
-            onChange={(value: string) => {
-                dispatch(changeApproversSubjectFilter({approversSubject: value}));
-            }}
-            className={block('subject-filter')}
-            value={subjectFilter}
-            size="m"
+        <Toolbar
+            itemsToWrap={[
+                {
+                    node: (
+                        <Filter
+                            placeholder="Filter by subject"
+                            onChange={(value: string) => {
+                                dispatch(changeApproversSubjectFilter({approversSubject: value}));
```

Comment:
It’s a little confusing that a new function will be generated here every time. But if it is not redrawn often, then you can leave it like this

Reply:
The component is used the same as before